package.json

```diff
@@ -15,6 +15,7 @@
   "devDependencies": {
     "babel-core": "^6.9.1",
     "babel-loader": "^6.2.4",
+    "babel-preset-es2015": "^6.9.0",
```

Comment:
This preset library was missing from package.json
When checked-out on fresh machine it prevented webpack to compile es2015 example